rfcs/text/0002-type-ann-for-struct-properties-in-structs.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: type-annotations-for-struct-properties-in-structs
+- Start Date: TODO
+- RFC PR: (leave this empty)
+- Feature Commit(s): (leave this empty)
+
+# Summary
+
+Typecheck a value expression for a struct property in a struct definition with
+expected type information in the property.
+
+# Motivation
+By adding types for struct properties, we further complete the type annotations
+for struct types.
+
+# Guide-level explanation
```

Comment:
This is (IMO) a potentially tricky subject with all the subtle details. I played around with the guide-level wording you had to try and remove potential ambiguities. I'm just going to insert them below the associated paragraph; please feel free to completely ignore or use any of them as you see fit!